hydrus/metadata/drone_doc_gen.py

```diff
@@ -14,15 +14,15 @@ def drone_doc(API, BASE_URL):
                        BASE_URL)
 
     # Status Class
-    status = HydraClass("Status", "Status", "Class for drone status objects")
+    status = HydraClass("http://hydrus.com/Status", "Status", "Class for drone status objects")
```

Comment:
We need a placeholder url to be able to use hydra-py. It uses a schema namespace to find operations we can't do that with vocab: status. Same goes for all.